CI coverage broke

<!-- Please fill in the following information to report a problem with Cantera. If you have a question about using Cantera, please post it on our Google Users' Group (https://groups.google.com/forum/#!forum/cantera-users). Feature enhancements should be discussed in the dedicated Cantera enhancements repository (https://github.com/Cantera/enhancements/new/choose) -->

**Problem description**

Coverage runner fails with
```
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/n/nvidia-settings/libxnvctrl0_460.39-0ubuntu0.20.04.1_amd64.deb  404  Not Found [IP: 52.147.219.192 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.
```

<!-- A clear and concise description of what the bug is. -->

**Steps to reproduce**

<!-- A minimal, complete, and reproducible example demonstrating the problem. See https://stackoverflow.com/help/minimal-reproducible-example for additional suggestions on how to create such an example. -->

See recent [GH Actions](https://github.com/Cantera/cantera/actions) failures

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -34,7 +34,9 @@ jobs:
         python-version: ${{ matrix.python-version }}
         architecture: x64
     - name: Install Apt dependencies
-      run: sudo apt-get install libboost-dev gfortran
+      run: |
+        sudo apt update
+        sudo apt install libboost-dev gfortran
     - name: Upgrade pip
       run: python3 -m pip install -U pip setuptools wheel
     - name: Install Python dependencies
@@ -106,7 +108,8 @@ jobs:
         architecture: x64
     - name: Install Apt dependencies
       run: |
-        sudo apt-get install libboost-dev gfortran liblapack-dev libblas-dev libsundials-dev
+        sudo apt update
+        sudo apt install libboost-dev gfortran liblapack-dev libblas-dev libsundials-dev
     - name: Upgrade pip
       run: python3 -m pip install -U pip setuptools wheel
     - name: Install Python dependencies
@@ -141,7 +144,9 @@ jobs:
           python-version: '3.8'
           architecture: x64
       - name: Install Apt dependencies
-        run: sudo apt-get install libboost-dev doxygen graphviz
+        run: |
+          sudo apt update
+          sudo apt install libboost-dev doxygen graphviz
       - name: Upgrade pip
         run: python3 -m pip install -U pip setuptools wheel
       - name: Install Python dependencies
@@ -189,7 +194,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: x64
       - name: Install Apt dependencies
-        run: sudo apt-get install libboost-dev gfortran graphviz liblapack-dev libblas-dev
+        run: |
+          sudo apt update
+          sudo apt install libboost-dev gfortran graphviz liblapack-dev libblas-dev
       - name: Upgrade pip
         run: python3 -m pip install -U pip setuptools wheel
       - name: Install Python dependencies
@@ -262,7 +269,9 @@ jobs:
         python-version: '3.8'
         architecture: x64
     - name: Install Apt dependencies
-      run: sudo apt-get install libboost-dev liblapack-dev libblas-dev
+      run: |
+        sudo apt update
+        sudo apt install libboost-dev liblapack-dev libblas-dev
     - name: Upgrade pip
       run: python3 -m pip install -U pip setuptools wheel
     - name: Install Python dependencies

